sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/WriteToDataSourceV2Exec.scala

```diff
@@ -76,9 +78,9 @@ case class CreateTableAsSelectExec(
 
       throw new TableAlreadyExistsException(ident)
     }
-
```

Comment:
Nit: I like having extra lines so code isn't so dense. Also, this isn't a functional change and could cause conflicts.